src/gui/wizard/webview.cpp

```diff
@@ -211,6 +218,15 @@ bool WebEnginePage::certificateError(const QWebEngineCertificateError &certifica
     return ret == QMessageBox::Yes;
 }
 
+bool WebEnginePage::acceptNavigationRequest(const QUrl &url, QWebEnginePage::NavigationType /*type*/, bool /*isMainFrame*/)
```

Comment:
I would prefer to leave the argument names as is and use  `Q_UNUSED` macro
seems to me better because some IDE will complain that the signature does not math the declaration in the header